client/lib/directly/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+// import { loadScript } from 'lib/load-script';
+
+const DIRECTLY_SCRIPT_PATH = 'https://www.directly.com/widgets/rtm/embed.js';
+const DEFAULT_RTM_WIDGET_OPTIONS = {
+	// Automattic widget ID
+	id: '8a2968fc57d1e2f40157f42bf2d43160',
+	// By default don't show the "Ask a Question" button
+	displayAskQuestion: false
+};
+
+let initialized = false;
+
+const loadDirectlyScript = ( callback = noop ) => {
+	// It would be great to use lib/load-script here, however the Directly script requires
+	// the <script> tag to have id="directlyRTMScript", otherwise it fails hard. So this is
+	// a stripped-down version of lib/load-script with the id added.
+	function handleCompletedRequest( event ) {
+		let errorArgument = null;
+		if ( this.readyState && this.readyState !== 'complete' ) {
+			return;
+		}
+		if ( event.type === 'error' ) {
+			errorArgument = { src: event.target.src };
+		}
+		callback( errorArgument );
+		this.onload = this.onreadystatechange = this.onerror = null;
+	}
+
+	const script = document.createElement( 'script' );
+	script.id = 'directlyRTMScript';
+	script.src = DIRECTLY_SCRIPT_PATH;
+	script.async = 1;
+	script.onload = script.onreadystatechange = script.onerror = handleCompletedRequest;
+	document.getElementsByTagName( 'head' )[ 0 ].appendChild( script );
+};
+
+const directly = {
+	initialize: ( config = {}, callback = noop ) => {
+		// Directly config and script loading can only happen once per pageload.
+		if ( initialized ) {
+			return;
+		}
+		initialized = true;
```

Comment:
Hm, so I put this flag internally in the module as a safety. It hard-prevents Directly from being re-initialized (we probably only want to execute the vendor script once, and even calling `DirectlyRTM('config')` after the vendor script loads will throws an error). It also prevents the other exported functions from doing anything until Directly has been initialized (perhaps these could warn in dev mode even).

Do you think those are reasonable safeties to put in place, and if so does this mean I should make this module Redux-aware?